app/views/beatboxer_tavily_search.py

```
@@ -3,8 +3,10 @@
 
 from flask import jsonify, request
 
+from app.models.gemini_client import gemini_service
```

Ruff (0.12.2)

`app.models.gemini_client.gemini_service` imported but unused

Remove unused import: `app.models.gemini_client.gemini_service`

(F401)